lib/content_block_tools/presenters/base_presenter.rb

```diff
@@ -36,13 +36,34 @@ def render
     private
 
       # The default representation of the content block - this can be overridden in a subclass
+      # by overriding the default_content method, or content_for_fields
       #
       # @return [string] A representation of the content block to be wrapped in the base_tag in
       # {#content}
       def content
+        if field_names.present?
+          content_for_fields
```

Comment:
i'm not sure the base presenter here should be processing field names, or if it should always only return title, but this was only way to test method to dig into nested fields worked without create a whole new type of block 🤔

Reply:
I think the way I planned it originally was to have a presenter for each block type, but if we're having a more field-based approach, this might not be the right move. I think this is fine for now - as we experiment, we can tidy this up a bit more.